### DISCUSSION TITLE
所以MusicFree协议用的是GPL还是AGPL

### DISCUSSION THREAD

**Discussion by OrO-c** [maotoumao/MusicFree#312] [Q&A]
![Screenshot_20240922142549](https://github.com/user-attachments/assets/203e78db-f2df-47e9-81c3-1ee06c3958d5)
![Screenshot_20240922142602](https://github.com/user-attachments/assets/2d6a3cef-cdf7-48bd-85be-86d29945ee7d)

**Comment by maotoumao** [comment]
Updated 前不久更新成了 AGPL

**Reply by OrO-c** [reply]
哦哦好
